packages/codegen-utils/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@latticexyz/codegen-utils",
```

Comment:
nit - should we call this package `common-codegen`? (Also wanna rename the `utils` package to `common`, so it makes more sense for it to also include shared types etc in addition to "utils" in the sense of utility functions)